tests/run-tests.sh

```diff
@@ -283,6 +324,32 @@ then
     CHECK_AOCL=0
     CHECK_NCCL=1
     CHECK_GCC=0
+elif [[ $distro == "SUSE Linux Enterprise High Performance Computing 15 SP4" ]]
+then
+    MKL_VERSION="2022.2.0"
+    #
+    CHECK_GCC=0
+    CHECK_ONEAPI=1
+    CHECK_AZURE_HPC_DIAG=1
+
+    HPCX_OMB_PATH=${HPCX_OMB_PATH_SUSE}
+    CHECK_HPCX=1
+    CHECK_IMPI_2021=1
+    CHECK_MVAPICH2=1
+    CHECK_MVAPICH2X=0
+    CHECK_CUDA=1
+    CHECK_OMPI=1
+    CHECK_BLIS_MT=1
+
+    MODULE_FILES_ROOT=${MODULE_FILES_ROOT_SUSE}
+    MOFED_VERSION=${MOFED_VERSION_SUSE}
+    IMPI2021_PATH=${IMPI2021_PATH_SUSE}
+    MVAPICH2_PATH=${MVAPICH2_PATH_SUSE}
+    OPENMPI_PATH=${OPENMPI_PATH_SUSE}
+
+    CHECK_AOCL=1
+    CHECK_NV_PMEM=0
+    CHECK_NCCL=0
```

Comment:
I believe nccl is being installed, any reason why it is off?

Reply:
oops, i added nccl in an second changeset and forgot to enable the test.